CDP4SiteDirectory/Views/Dialogs/PersonDialog.xaml

```diff
@@ -145,6 +145,9 @@
                     </Grid>
 
                 </lc:LayoutItem>
+                <items:WarningLayoutItem Visibility="{Binding Path=IsSelectedRoleDeprecated, UpdateSourceTrigger=PropertyChanged,
+                    Converter={StaticResource BooleanToVisibilityConverter }}"
+                                         WarningText="The selected role is deprecated."/>
```

Comment:
We are used to use the 10-25 terminology in user text. So here it would be "PersonRole"instead of "role".